### PR TITLE
Visuals remove scrolling + show number labels

### PR DESCRIPTION
**Pull request by karimkawambwa**
- the issue with scaling will have to be resolved in hurumap ui https://github.com/CodeForAfrica/HURUmap-UI/pull/41

Screen Shot 2019-08-20 at 13 07 37

### CODE REVIEW

**Comment by karimkawambwa** [comment]
@KhadijaMahanga can you give me a quick review?

File: src/pages/Profile.js [hunk 1]
```diff
@@ -279,20 +290,10 @@ query charts($geoCode: String!, $geoLevel: String!) {
...
-                overflowY={
-                  chart.visuals.find(visual => visual.type === 'pie')
-                    ? 'visible'
-                    : chart.visuals.find(visual => visual.horizontal)
-                    ? 'auto'
-                    : 'hidden'
-                }
+                /* TODO: hurumap-ui remove scroll */
+                overflowX="visible"
+                overflowY="visible"
```
Comment (kilemensi) [inline src/pages/Profile.js]: Should these still be here?

File: src/components/Hero/ProfileHero/Profile.js [hunk 2]
```diff
@@ -153,20 +153,35 @@ function Profile({ classes, dominion, geoId, history, ...props }) {
...
         }
 
         let population;
         let populationDensity;
         if (totalPopulation) {
-          population = totalPopulation.toFixed(0);
-          populationDensity = (totalPopulation / squareKms).toFixed(2);
+          let numberFormatter = new Intl.NumberFormat('en-IN');
+          population = numberFormatter.format(totalPopulation.toFixed(0));
+          numberFormatter = new Intl.NumberFormat('en-IN', {
```
Comment (kilemensi) [inline src/components/Hero/ProfileHero/Profile.js]: This number format can be reused I think

**Comment by kilemensi** [comment]
And I'm assuming there we've agreed to completely remove tooltips from bar charts @karimkawambwa.

File: src/components/Hero/ProfileHero/Profile.js (hunk 2 above)
Comment (karimkawambwa) [reply src/components/Hero/ProfileHero/Profile.js]: There is no way to reset the options @kilemensi

File: src/pages/Profile.js (hunk 1 above)
Comment (karimkawambwa) [reply src/pages/Profile.js]: Am waiting for the new `hurumap-ui` to be published then all these will be gone.

**Comment by karimkawambwa** [comment]
@kilemensi 

> Some of the bar chart width-to-height ratio is a bit off

I can't help that, with the auto scaling that's automatically going to make the chart that weird since we can't scroll and so the scaling is taking over. Setting responsive off will make the chart a good size but its going to be cut off on the sides.

The bar width will be reduced once you merge in the new `hurumap-ui` and maybe we will have to customize each charts bar width so a new variable barWidth in chart.json. We will have to probably rotate the labels swell if the bar is too thin or convert it to horizontal. 

cc @DavidLemayian @KhadijaMahanga 

> And I'm assuming there we've agreed to completely remove tooltips from bar charts

This was not something discussed 

> Nested area numbers are not formatted. Is this intentional?

I did not look at that.
hopefully I can axes that label datum otherwise it will have to be done within `hurumap-ui` or expose the datum.

**Comment by kilemensi** [comment]
>>   And I'm assuming there we've agreed to completely remove tooltips from bar charts

> This was not something discussed

I was specifically asking about these lines here @karimkawambwa. 

https://github.com/CodeForAfrica/Dominion.AFRICA/blob/1491bb7266d55f2755a0a8172bf4cfa71204f98d/src/components/ChartFactory.js#L158-L159

**Comment by karimkawambwa** [comment]
@kilemensi without setting undefined, the label won't show up. I did not want a tooltip and I don't believe/know if column tooltips is something that is need to support.

**Comment by karimkawambwa** [comment]
@kilemensi I would like to merge this and then when `hurumap-ui` is published, the changes will be updated. What do you think?

File: src/components/Hero/ProfileHero/Profile.js (hunk 2 above)
Comment (kilemensi) [reply src/components/Hero/ProfileHero/Profile.js]: I meant you could reuse the one declared on L167 above. It's exactly as this one but it's no biggie.

**Comment by kilemensi** [comment]
> @kilemensi I would like to merge this and then when `hurumap-ui` is published, the changes will be updated. What do you think?

Sounds like a plan @karimkawambwa ... We probably need to implement a custom label component in HURUmap UI first anyways so go ahead merge this and create a PT story for the custom label component which should then be the default label component on bar chart in HURUmap UI.

**Comment by karimkawambwa** [comment]
@kilemensi custom label component to support tooltip and label ?

**Pull request merged**